hash/neogeo.xml

```diff
@@ -11436,6 +11436,94 @@ license:CC0-1.0
+				<rom name="looptris.s1" offset="0x000000" size="0x020000" crc="70e70448" sha1="40461af534afcba341d2f14b755d6de353fe7afd" />
+			</dataarea>
+			<dataarea name="audiocpu" size="0x020000">
+				<rom name="looptris.m1" offset="0x000000" size="0x020000" crc="e7105df8" sha1="9f4efad6dcc8b9617d59576415019820e973f45c" />
+			</dataarea>
+			<dataarea name="ymsnd:adpcma" size="0x80000">
+				<rom name="looptris.v1" offset="0x000000" size="0x80000" crc="dfa63cd2" sha1="bbf4a74a660f12cc037cfa0ac5c664988016ee45" />
+			</dataarea>
+			<dataarea name="sprites" size="0x100000">
+				<rom loadflag="load16_byte" name="looptris.c1" offset="0x000000" size="0x80000" crc="b9413f13" sha1="5e7175dff9510dad55f8f05e9b307c3c05d92209" />
+				<rom loadflag="load16_byte" name="looptris.c2" offset="0x000001" size="0x80000" crc="9409dbe8" sha1="1e9562aa34f6202e8012104ab79916453fec072e" />
+			</dataarea>
+		</part>
+    </software>
 
+	<software name="looptrsp">
+        <description>Looptris Plus</description>
+		<year>2022</year>
+		<publisher>Blastar</publisher>
+		<info name="release" value="20221224" />
+		<sharedfeat name="release" value="AES" />
+		<sharedfeat name="compatibility" value="MVS,AES" />
+		<part name="cart" interface="neo_cart">
+			<dataarea name="maincpu" width="16" endianness="big" size="0x80000">
+				<rom loadflag="load16_word_swap" name="looptrsp.p1" offset="0x000000" size="0x80000" crc="894bb290" sha1="f33ed9fe75ca7b5a6da00fe5bff76e26cba85e08" />
+			</dataarea>
+			<dataarea name="fixed" size="0x020000">
+				<rom offset="0x000000" size="0x020000" name="looptrsp.s1" crc="70e70448" sha1="40461af534afcba341d2f14b755d6de353fe7afd" />
+			</dataarea>
+			<dataarea name="audiocpu" size="0x020000">
+				<rom offset="0x000000" size="0x020000" name="looptrsp.m1" crc="249bba11" sha1="e8248ff9bc391b1163746c256b137928a9d81fe0" />
+			</dataarea>
+			<dataarea name="ymsnd:adpcma" size="0x200000">
+				<rom name="looptrsp.v1" offset="0x000000" size="0x080000" crc="c9f86637" sha1="edb712d5cce180e1d5b6e338e9abfab1125c5581" />
+				<rom name="looptrsp.v2" offset="0x080000" size="0x080000" crc="41b3e17a" sha1="ef216379dbdbbea04cfffa47123436213a8cf26e" />
+				<rom name="looptrsp.v3" offset="0x100000" size="0x080000" crc="1ed4e538" sha1="921913be499194236535052ed3140fb7169d16e9" />
+				<rom name="looptrsp.v4" offset="0x180000" size="0x080000" crc="705e7065" sha1="a5eea38afa58db69c0f17e0379da3d819639971f" />
+			</dataarea>
+			<dataarea name="sprites" size="0x100000">
+				<rom loadflag="load16_byte" name="looptrsp.c1" offset="0x000000" size="0x80000" crc="b9413f13" sha1="5e7175dff9510dad55f8f05e9b307c3c05d92209" />
+				<rom loadflag="load16_byte" name="looptrsp.c2" offset="0x000001" size="0x80000" crc="9409dbe8" sha1="1e9562aa34f6202e8012104ab79916453fec072e" />
+			</dataarea>
+		</part>
+    </software>
+
+	<software name="neotris">
+		<description>NeoTRIS</description>
+		<year>2020</year>
+		<publisher>Chips on Steroids</publisher>
```

Comment:
Should this be partially supported, as the four-player adapter is probably incompatible with the Kizuna Encounter one?

Reply:
Demoted do "partial"